sportnet/sportnet/playgrounds/models.py

```diff
@@ -15,3 +15,15 @@ class Meta:
                                     related_name='own_playgrounds')
     players = models.ManyToManyField(get_user_model(), verbose_name='participants',
                                      related_name='in_playgrounds')
+
+
+class Sport(models.Model):
+    class Meta:
+        verbose_name = 'Kind of sport'
```

Comment:
Лучше на русском делать verbose_name. Целевая аудитория у нас все таки из РФ.
```suggestion
        verbose_name = 'Вид спорта'
```